_layouts/product.html

```diff
@@ -183,6 +183,18 @@
   <td {% if diff <= 0 %} class = "txt-linethrough" {% endif %} >      <!-- if the support finished add txt-linethrough class -->
     {% if releaseLink != "" %}
       <a href="{{releaseLink}}" title="Release Notes / Changelog">{{latestVersionNumber}}</a>
+{% if  diff > 1 %}
+      &ensp;
+      <a href="http://www.google.com/calendar/render?action=TEMPLATE
+&text=[endoflife.date] {{page.title}} {{r.releaseCycle}} EOL reached
+&dates={{r.eol | replace:'-',''}}T000000/{{r.eol | replace:'-',''}}T000000
+&details=It will be end of support for  {{page.title}} {{r.releaseCycle}}
+&trp=false
+&sprop=
+&sprop=name:" target="_blank" rel="nofollow">
```

Comment:
A title on the link (such as `title="Add to Calendar"`) could be useful.

Reply:
Actually i did it but mistakenly on my local part and forget to add it :D